### DISCUSSION TITLE
Almost the same color of road and route

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20415] [General]
![Screenshot_2024-07-25-21-53-20-362_net osmand plus](https://github.com/user-attachments/assets/6e4868e3-6e92-4619-a884-7af26a3208e4)

This wasn't supposed to be a note of the user!

**Comment by DmitryAlexei** [comment]
You can customize route line color, please see https://osmand.net/docs/user/navigation/guidance/navigation-settings#customize-route-line Please note, that the route line is blue by default
You can also change map style https://osmand.net/docs/user/map/vector-maps#default-map-styles
